app/assets/stylesheets/user-styles.css

```diff
@@ -35,3 +35,11 @@ img.right {
 #sidebar .sidebar-body li {
   list-style-type: circle;
 }
+
+#archives{
+    list-style: circle;
+}
+
+.sidebar_title {
+  font-family: monospace;
```

Comment:
This PR introduces redundant CSS rules for styling the sidebar. Can we use existing rules?